cmd/servicelog/post.go

```diff
@@ -43,7 +43,7 @@ const (
 
 // postCmd represents the post command
 var postCmd = &cobra.Command{
-	Use:   "post",
+	Use:   "post CLUSTER_ID",
```

Comment:
I think C0bra has an automated way to generate this first argument. You might want to look for "Positional arguments" with c0bra :thinking:

Reply:
Indeed this would make sense and be better

Reply:
I based my work on this example found in Cobra code base:
https://github.com/spf13/cobra/blob/master/doc/cmd_test.go
In which they do not use positional args... but they exist, so lets use them. Thanks!

Reply:
Hum, actually even if it is possible to give constraints on the number of arguments, it is not possible to name them:
https://stackoverflow.com/questions/69167939/named-positional-arguments-in-cobra
Here the API to constraint the number of args (something we could use as at most one cluster id is allowed):
https://github.com/spf13/cobra/blob/45e521ea93e2270ef7333487f0a87e3887de2da9/args.go#L60

This is also discussed there:
https://github.com/spf13/cobra/issues/395#issuecomment-293634704